images.py

```diff
@@ -251,6 +269,36 @@ def _purge():
             session.query(Link).delete()
 
 
+STATICMAP_URL = (
+    "https://maps.googleapis.com/maps/api/staticmap?"
+    "zoom={0}&style=feature:poi%7Cvisibility:off"
+    "&size={1}x{2}&language=is&scale=2&maptype=roadmap"
+    "&key={3}&markers={4},{5}"
+)
+
+
+def get_staticmap_image(latitude, longitude, zoom=6, width=180, height=180):
+    """ Request image from Google Static Maps API, return image data as bytes """
+    key = _get_API_key()
+    if not key:
+        return None
+
+    url = STATICMAP_URL.format(zoom, width, height, key, latitude, longitude)
+    # TODO: Use urllib instead of requests here
```

Comment:
I'm not sure how Gunicorn/gevent/eventlet handle this, i.e. whether a call to `requests` blocks the web server worker. I suspect it does.

Reply:
Yeah, it blocks the worker. Do you have any suggestions on how to handle this differently? There's a variety of async requests-like libraries out there.

Reply:
Need to investigate whether using urllib would be compatible with the monkeypatching scheme of greenlets/eventlets.

Reply:
...but the futureproof solution is to migrate to an async based web server framework, such as `responder`.